src/pipeline/pipeline.py

```diff
@@ -42,7 +42,7 @@
 
 total_sightings_found = 0
 db_ingest_count = 0
-db_ingest_batch_size = 1000
+db_ingest_batch_size = os.getenv("db_ingest_batch_size", 1000)
```

Comment:
this is a ~cleaner way to process environment variables than accessing the os environment dictionary. Provides a default value out of the gate :)